docs/source/nn/index.rst

```diff
@@ -0,0 +1,26 @@
+.. _nn-index:
+
+torch.nn
+===================================
+
+.. toctree::
+   :maxdepth: 1
+
+   nn_parameters.rst
```

Comment:
Is there a reason why we chose to split the nn documentation into sections as opposed to having one nn function per page? (I'm not advocating for one over the other, just curious about what we've considered)

Reply:
I broke them up according to the current breakdown (https://github.com/pytorch/pytorch/blob/master/docs/source/nn.rst). Open to other options if those are more intuitive.

Reply:
I think this is reasonable for now